packages/inventory/src/Stocktake/ListView/ListView.tsx

```diff
@@ -64,30 +48,16 @@ export const StocktakeListView: FC = () => {
     [sortBy]
   );
 
-  const onNewStocktake = async () => {
-    try {
-      const id = generateUUID();
-      const result = await onCreate({ id });
-      invalidate();
-      navigate(result);
-    } catch (e) {
-      const errorSnack = error(
-        'Failed to create stocktake! ' + (e as Error).message
-      );
-      errorSnack();
-    }
-  };
-
   return (
     <>
-      <Toolbar onDelete={onDelete} data={data} filter={filter} />
-      <AppBarButtons onCreate={onNewStocktake} />
+      <Toolbar onDelete={() => {}} data={data?.nodes} filter={filter} />
```

Comment:
will create a delete hook as a specific issue - really just wanting to get into a compiling state